ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,50 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_test
+
+
+# result_type
+@handle_test(
```

Comment:
I think it's better to use the `@handle_frontend_test` decorator. Kindly refer to other examples in the frontend to see how it's been used.